### PR TITLE
Add "Key Value" service image link in "Sign up for the Ad Selection API"

### PR DESCRIPTION
**Pull request by mikehoffms**
Rendered article sections for review:

* **Sign up for the Ad Selection API** article
   * **Images for deploying services** section
      * https://review.learn.microsoft.com/microsoft-edge/web-platform/ad-selection-api?branch=pr-en-us-3287#images-for-deploying-services
      * [Before/Live](https://learn.microsoft.com/microsoft-edge/web-platform/ad-selection-api#images-for-deploying-services)
      * Added list item for buyers server-image for "Bidding Selection & Key/Value" service.
   * **See also** section
      * https://review.learn.microsoft.com/microsoft-edge/web-platform/ad-selection-api?branch=pr-en-us-3287#see-also
      * [Before/Live](https://learn.microsoft.com/microsoft-edge/web-platform/ad-selection-api#see-also)
      * Added list item for buyers server-image for "Bidding Selection & Key/Value" service.

To approve the PR:
**Files changed** page > **Review changes** green button > **Approve** option button + comment "LGTM - Dev|PM"

AB#54374954

### CODE REVIEW

File: microsoft-edge/web-platform/ad-selection-api.md
Comment (mikehoffms) [inline microsoft-edge/web-platform/ad-selection-api.md:1]: change title at
https://mcr.microsoft.com/product/ad-selection/azure/key-value-service/
from:
Ad Selection Key Value Service
to:
Ad Selection Key/Value Service
per names of the services as listed in sections: 
https://learn.microsoft.com/microsoft-edge/web-platform/ad-selection-api#services-for-sellers

and then change link text in present article 2x

@yanushP

Comment (mikehoffms) [inline microsoft-edge/web-platform/ad-selection-api.md:1]: add link for "Bidding Selection & Key/Value" in this PR?
@yanushP

**Review by yanushP — APPROVED** [review]

File: microsoft-edge/web-platform/ad-selection-api.md
Comment (mikehoffms) [reply microsoft-edge/web-platform/ad-selection-api.md:1]: also you could increase clarity of titles of the service images pages by changing from "Service" to "service", leveraging the industry's move to "Sentence casing" of titles and headings per corp style guide: 
https://learn.microsoft.com/en-us/style-guide/capitalization -- "Use sentence-style capitalization ... Capitalize the first word of a sentence, heading, title, UI label (such as the name of a button or checkbox), or standalone phrase."
@yanushP

File: microsoft-edge/web-platform/ad-selection-api.md [hunk 1]
```diff
@@ -170,14 +170,16 @@ Each image defines one service.  Some images don't contain UDFs, and consist ent
...
-* [Ad Selection Seller Front End Service](https://mcr.microsoft.com/product/ad-selection/azure/seller-frontend-service)
-* [Ad Selection Auction Service](https://mcr.microsoft.com/product/ad-selection/azure/auction-service)
-
-Images for buyers:
-* [Ad Selection Buyer Front End Service](https://mcr.microsoft.com/product/ad-selection/azure/buyer-frontend-service)
-* [Ad Selection Bidding Service](https://mcr.microsoft.com/product/ad-selection/azure/bidding-service)
-* [Ad Selection K-Anonymity Service](https://mcr.microsoft.com/product/ad-selection/azure/k-anonymity-service)
+Images for deployment:
+* Sellers
+   * [Ad Selection Seller Front End Service](https://mcr.microsoft.com/product/ad-selection/azure/seller-frontend-service)
```
Comment (mikehoffms) [inline microsoft-edge/web-platform/ad-selection-api.md:175]: maybe for future PR:
in service-image page titles and in present article globally, make name of services consistent re: space chars, cap'zn, & bold where applicable.  eg everywhere, say either:
* the **Seller Front End** service
* the **SellerFrontEnd** service
@yanushP @captainbrosset

**Review by captainbrosset — APPROVED** [review]

File: microsoft-edge/web-platform/ad-selection-api.md (hunk 1 above)
Comment (mikehoffms) [reply microsoft-edge/web-platform/ad-selection-api.md:175]: also check usage in articles (how the name of service is presented/formatted) about Ad Selection API in other repo.
@yanushP @captainbrosset

**Review by mikehoffms — COMMENTED** [review]
LGTM - Writer/Editor

**Pull request merged**